python/ray/tune/analysis/test_experiment_analysis.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import unittest
+import shutil
+import random
+import os
+import pandas as pd
+
+from experiment_analysis import ExperimentAnalysis
+import ray
+from ray.tune import run, sample_from, Trainable
+from ray.tune.examples.async_hyperband_example import MyTrainableClass
+from ray.tune.schedulers import AsyncHyperBandScheduler
+
+
+class ExperimentAnalysisSuite(unittest.TestCase):
+    def setUp(self):
+        ray.init()
+
+        self.test_dir = "~/analysis_test"
```

Comment:
let's do ` tempfile.mkdtemp()` instead; it'll give you a temporary directory (and you can shutil.rmtree it too)